requirements.txt

```diff
@@ -15,6 +15,7 @@ coverage==5.5
 cryptography==3.4.7
 defusedxml==0.7.1
 dictpath==0.1.3
+dpath==2.0.6
```

Comment:
Kun je deze ook in de requirements-root.txt plaatsen? Zie ook https://github.com/Amsterdam/mijn-wmoned-api/blob/main/requirements-root.txt#L3